File: src/api/utility/Requests.cs
using System;
using System.Collections.Generic;
using NetTopologySuite.Geometries;
using DVAN.Population;

namespace DVAN.API
{
    /// <summary>
    /// Request to store population view (from given parameters).
    /// </summary>
    public class PopulationStoreRequest
    {
        public PopulationRequestParams population { get; set; }
    }

    /// <summary>
    /// Request to get data from stored or internal population view.
    /// </summary>
    public class PopulationGetRequest
    {
        /// <summary>
        /// Population view id.
        /// </summary>
        /// <example>sfjf-djfd-omsf-jjfd</example>
        public Guid? population_id { get; set; }

        public PopulationRequestParams? population { get; set; }
    }
}

File: src/main/java/org/tud/oas/api/utility/Requests.java
package org.tud.oas.api.utility;

import java.util.UUID;

import org.tud.oas.demand.DemandRequestParams;

import com.fasterxml.jackson.annotation.JsonProperty;

/// <summary>
/// Request to store population view (from given parameters).
/// </summary>
class PopulationStoreRequest {
    @JsonProperty("population")
    public DemandRequestParams population;
}

/// <summary>
/// Request to get data from stored or internal population view.
/// </summary>
class PopulationGetRequest {
    /// <summary>
    /// Population view id.
    /// </summary>
    /// <example>sfjf-djfd-omsf-jjfd</example>
    @JsonProperty("population_id")
    public UUID population_id;

    @JsonProperty("population")
    public DemandRequestParams population;
}
